ci/pipeline.yml

```diff
@@ -0,0 +1,151 @@
+---
+jobs:
+- name: push-cg-quotas-db-staging
+  plan:
+  - get: cg-quotas-db-staging
+    trigger: true
+  - task: run-tests
+    file: cg-quotas-db-staging/ci/run-tests.yml
+    input_mapping:
+      cg-quotas-db: cg-quotas-db-staging
+  - task: create-db
+    file: cg-quotas-db-staging/ci/create-db.yml
+    input_mapping:
+      cg-quotas-db: cg-quotas-db-staging
+    params:
+      SERVICE_NAME: rds-cg-quotas-db
+      SERVICE_PLAN: shared-psql
+      CF_API_URL: {{staging-cf-api-url}}
+      CF_DEPLOY_USERNAME: {{staging-cf-deploy-username}}
+      CF_DEPLOY_PASSWORD: {{staging-cf-deploy-password}}
+      CF_ORGANIZATION: {{staging-cf-organization}}
+      CF_SPACE: {{staging-cf-space}}
+  - put: deploy-cg-quotas-db-staging
+    params:
+      manifest: cg-quotas-db-staging/manifest.yml
+      path: cg-quotas-db-staging
+      current_app_name: quotas
+      environment_variables:
+        APP_SETTINGS: config.ProductionConfig
+        CF_API_URL: {{staging-cf-api-url}}
+        CF_UAA_URL: {{staging-cf-uaa-url}}
+        CF_USERNAME: {{staging-cf-uaa-username}}
+        CF_PASSWORD: {{staging-cf-uaa-password}}
+        SECRET_KEY: {{staging-secret-key}}
+        SECRET_USERNAME: {{staging-basic-auth-username}}
+        SECRET_PASSWORD: {{staging-basic-auth-password}}
+    on_failure:
+      put: slack
+      params:
+        text: |
+          :x: FAILED to deploy cg-quotas-db on {{staging-cf-api-url}}
+          <https://ci.cloud.gov/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|View build details>
+        channel: {{slack-channel}}
+        username: {{slack-username}}
+        icon_url: {{slack-icon-url}}
+    on_success:
+      put: slack
+      params:
+        text: |
+          :white_check_mark: Successfully deployed cg-quotas-db on {{staging-cf-api-url}}
+          <https://ci.cloud.gov/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|View build details>
+        channel: {{slack-channel}}
+        username: {{slack-username}}
+        icon_url: {{slack-icon-url}}
+
+- name: push-cg-quotas-db-prod
+  plan:
+  - get: cg-quotas-db-prod
+    trigger: true
+  - task: run-tests
+    file: cg-quotas-db-prod/ci/run-tests.yml
+    input_mapping:
+      cg-quotas-db: cg-quotas-db-prod
+  - task: create-db
+    file: cg-quotas-db-prod/ci/create-db.yml
+    input_mapping:
+      cg-quotas-db: cg-quotas-db-prod
+    params:
+      SERVICE_NAME: rds-cg-quotas-db
+      SERVICE_PLAN: shared-psql
```

Comment:
I'm going to say yes, because:
* The aws broker takes 15-20 minutes to provision a new dedicated instance, and because it doesn't (yet) implement async broker operations, we have to keep trying to push the app, checking for a bind-related failure, and sleeping until it works.
* AFAICT, the database is basically a small cache of the cf api, meaning we can restore it as needed if anything gets lost.